CHANGELOG.md

```diff
@@ -16,6 +16,9 @@
   Serverless Functions plugins: it does not provide access to the global kong cache. Access to
   certain fields in kong.configuration has also been restricted.
   [#10417](https://github.com/Kong/kong/pull/10417)
+- **Opentelemetry**: plugin version has been updated to match Kong's version
+  [#10646](https://github.com/Kong/kong/pull/10646)
```

Comment:
was this intentionally put under "breaking changes"?

Reply:
@samugi Yes. A user may be surprised that the version has jumped.